CONTRIBUTING.md

```diff
@@ -19,15 +19,13 @@ This is a *guideline* which means that some exceptions may be made. However it i
 
 ### Requirements
 
-You will need python 3 and the library `discord.py`.
-
-*(We would also recommend to work on Linux :penguin:)*
+You will need python >= 3.7 and the library `discord.py`.
```

Comment:
should we provide a link to the repo or is it fine since it's already done in the README?